api/app/controllers/spree/api/checkouts_controller.rb

```diff
@@ -11,9 +11,6 @@ class CheckoutsController < Spree::Api::BaseController
 
       include Spree::Core::ControllerHelpers::Order
 
-      # TODO: Remove this after deprecated usage in #update is removed
-      include Spree::Core::ControllerHelpers::PaymentParameters
```

Comment:
This can't be removed because `update` uses `set_payment_parameters_amount`. I'm going to remove the comment about deprecation.

Reply:
Yes, makes sense. The comment didn't take that into account. 👍

Reply:
The method has been added later. I'm going to squash the commit `https://github.com/solidusio/solidus/pull/3818/commits/51597744f1291a828aaa3ef3c9d7f9544559c790`

Reply:
Seems like it [was already there](https://github.com/solidusio/solidus/commit/a173f23884ef3901ceef3f7fd4fb1dff1ae42223#diff-45fff904f0af543763fdd5e09e339a490d759b46b78fabb150bd0f41e532570bR91), but nevermind. We know what to do!